presto-hive/src/test/java/com/facebook/presto/hive/TestHiveLogicalPlanner.java

```diff
@@ -363,13 +363,40 @@ public void testPushdownArraySubscripts()
         assertPushdownSubfields("SELECT id, b[1] FROM test_pushdown_array_subscripts CROSS JOIN UNNEST(a) as t(unused)", "test_pushdown_array_subscripts",
                 ImmutableMap.of("b", toSubfields("b[1]")));
 
+        // Aggregation
+        assertPushdownSubfields("SELECT arbitrary(y[1]).a FROM test_pushdown_array_subscripts GROUP BY id", "test_pushdown_array_subscripts",
```

Comment:
nit: these tests can go into `assertPushdownSubscripts` which is shared between array and map tests; this way you won't need to copy paste them

Reply:
Done